### ISSUE TITLE
Possible Null exception loop when admin search is enabled before indexing service

### ISSUE THREAD

**Issue by Xeevis** [OrchardCMS/Orchard#5280]
When modules  **Admin Content Search** or **Media Library Search** are enabled before Lucene, they'll have null exception failing their migrations. If both are enabled after one another, site will end up in endless loop writing errors into log. This should be handled more gracefully, without creating hard dependency on Lucene.

Tested on `dev`.

**Comment by mcoulter567** [comment]
Did you find a solution to this? I'm experience this issue and get the following message for admin content search. 

Invalid search query: Value cannot be null. 
Parameter name: key

I tried disabling everything and then enabling them in the order you mentioned but still getting the null error.

Was working fine in previous versions. Unfortunately, I've placed 1.9 in production and didn't catch this during testing. Tried to revert back to 1.82 and
